gnovm/pkg/gnolang/realm.go

```diff
@@ -1517,10 +1518,14 @@ func isUnsaved(oo Object) bool {
 // be realms and as such to have their state persisted. This is used by [IsRealmPath].
 const realmPathPrefix = "gno.land/r/"
 
+var ReGnoRunPath = regexp.MustCompile(`gno\.land/r/g[a-z0-9]+/run`)
```

Comment:
We should change this to /u/ maybe, but we'll leave that For Another Time(tm)...